src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -773,7 +773,7 @@ public bool IsMethodLevelMember([NotNullWhen(true)] SyntaxNode? node)
 
         public bool IsTopLevelNodeWithMembers([NotNullWhen(true)] SyntaxNode? node)
         {
-            return node is NamespaceDeclarationSyntax ||
+            return node is BaseNamespaceDeclarationSyntax ||
```

Comment:
@CyrusNajmabadi There are other references to `NamespaceDeclarationSyntax` in this class, some of which I feel they should be adjusted to handle file-scoped namespaces. Are you tracking doing that?